modules/rosa-delete-users.adoc

```diff
@@ -0,0 +1,21 @@
+
+// Module included in the following assemblies:
+//
+// getting_started_rosa/rosa-creating-cluster.adoc
+
+
+[id="rosa-delete-users"]
+= Revoking administrator access using the OCM console
+You can revoke the `dedicated-admin` or `cluster-admin` access of users through the OpenShift Cluster Manager (OCM) console. Users will be able to access the cluster without administrator privileges.
+
+.Prerequisites
+
+* You have added an Identity Provider (IDP) to your cluster.
+* You have the IDP user name for the user whose privileges you are revoking.
+* You are logged in to the OCM console using the OCM account that was used to create the cluster, the organization administrator user, or the superadministator user.
```

Comment:
Also, I believe `super administrator` is two words.

Reply:
Made the change.